src/native/fs/fs_napi.cpp

```diff
@@ -254,11 +255,12 @@ struct Mkdir : public FSWorker
     Mkdir(const Napi::CallbackInfo& info) : FSWorker(info)
     {
         _path = info[1].As<Napi::String>();
+        // TODO: Handle mode
         Begin(XSTR() << DVAL(_path));
     }
     virtual void Work()
     {
-        int r = mkdir(_path.c_str(), S_IRWXU);
+        int r = mkdir(_path.c_str(), S_IRWXU | S_IRWXG | S_IRWXO);
```

Comment:
Are these all the 3 sticky bits for User/Group/Other ?
Can you add a comment to explain why we are using this mode?